doc/source/serve/ml-models.rst

```diff
@@ -64,6 +64,38 @@ That's it. Let's take a look at an example:
 
 .. literalinclude:: ../../../python/ray/serve/examples/doc/snippet_model_composition.py
 
+Integration with Model Registries
+=================================
+
+Ray Serve is flexible.  If you can load your model as a Python
+function or class, then you can scale it up and serve it with Ray Serve.
+
+For example, if you are using the 
+`MLflow Model Registry <https://www.mlflow.org/docs/latest/model-registry.html>`_
+to manage your models, the following wrapper
+class will allow you to load a model using its MLflow `Model URI`: 
+
+.. code-block:: python
+
+  import mlflow.pyfunc
+
+  class MLflowBackend:
+      def __init__(self, model_uri):
+          self.model = mlflow.pyfunc.load_model(model_uri=model_uri)
+
+      async def __call__(self, request):
+          df = await request.body() # pandas DataFrame
```

Comment:
how do you get this into a pandas dataframe? it's not very obvious here.
How about:
```suggestion
          csv_text = await request.body() # The body contains just raw csv text.
          df = pd.read_csv(csv_text)
```